FlowersStore/Scenes/FlowersList/FlowersListInteractor.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  FlowersListInteractor.swift
+//  FlowersStore
+//
+//  Created by Kauê Sales on 30/11/21.
+//
+
+import Foundation
+
+protocol FlowersListInteractorInput: AnyObject {
+    
+}
+
+protocol FlowersListInteractorOutput: AnyObject {
+    
+}
+
+class FlowersListInteractor {
+    private weak var output: FlowersListInteractorOutput?
+    
+    func setOutput(_ output: FlowersListInteractorOutput?) {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'setOutp...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowersListInteractor.swift;
  Line: 21;
-->

- :warning: Function 'setOutput(_:)' is unused